### ISSUE TITLE
How to prevent opening a browser window?

### ISSUE THREAD

**Issue by fdietze** [Globegitter/broccoli-browser-sync#4]
Everytime I run `broccoli serve`, a browser window opens. I'd like to disable it and and found that this is a BrowserSync option. But it doesn't work for me.
Here is my relevant Brocfile.js:

``` JavaScript
    var browserSync = new BrowserSync([styles, scripts], {
        // proxy the local play server
        port: 9000,
        browserSync: {
            open: false // should not open the browser
        }
    });
```

What am I doing wrong?

**Comment by cornerman** [comment]
This has not been released yet; please release a new version.

**Comment by mlaccetti** [comment]
Fixed in 1.0.1.
